protecode/scanning_util.py

```diff
@@ -377,41 +387,45 @@ def enumerate_reference_triages():
                 self._api.rescan(protecode_app.product_id())
 
         # upload new images
-        for resource in images_to_upload:
+        for upload_element in elements_to_upload:
             try:
                 scan_result = self._upload_image(
```

Comment:
nit: if thinking about renaming the variable names (image -> _artefact_), the method names ought also be renamed. to have a chance of eventually finalising this PR, please revert those name-changes back for the time being, so we can keep the focus on correcting the bug. do refactoring later.